.gitignore

```diff
@@ -8,3 +8,4 @@ test-results.xml
 __pycache__
 # generated by setuptools_scm
 bumble/_version.py
+.vscode/launch.json
```

Comment:
Maybe `.vscode/` without file ?

Reply:
we do want `.vscode/setttings` for the workspace settings where some basic linter settings are.